server/render/index.js

```diff
@@ -137,10 +144,11 @@ export function serverRender( req, res ) {
 	context.config = config.ssrConfig;
 
 	if ( config.isEnabled( 'desktop' ) ) {
-		res.render( 'desktop', context );
-	} else {
-		res.render( 'index', context );
+		renderJsxStream( 'desktop', context ).pipe( res );
+		return;
 	}
+
+	renderJsxStream( 'index', context ).pipe( res );
```

Comment:
Are we just streaming the template JSX here? What about SSRed sections? Is it worth splitting the streaming into separate PRs?

Reply:
Yes, I'm going to separate streaming as it's potentially a bigger change if we not just only stream the template JSX (like now) but also SSRed sections.